infrastructure/quick-deploy/localhost/all-in-one/variables.tf

```diff
@@ -95,7 +95,22 @@ variable "activemq" {
     node_selector      = optional(any, {})
     image_pull_secrets = optional(string, "")
   })
-  default = {}
+  default = null
+}
+
+# Parameters for RabbitMQ
+variable "rabbitmq" {
+  description = "Parameters of RabbitMQ"
+  type = object({
+    enable                = optional(bool, true)
+    namespace             = optional(string)
```

Comment:
For the other services, we did not expose the possibility to change the namespace of specific resources. We deploy everything in the same namespace (`armonik` by default).